### ISSUE TITLE
WCAG 1.3.1 A: List not nested (Issue 9)

### ISSUE THREAD

**Issue by CJE001** [civictheme/uikit#377]
### Summary

Via Vision Australia assessment: August 2024

Impact: low

Note: DTA have a 90 day remediation period to address the identified issues within the audit, all issues must be resolved to obtain WCAG 2.2 certification for digital.gov.au.

### Steps to reproduce

Global issue – Main navigation
[digital.gov.au beta | digital.gov.au](https://www.digital.gov.au/) (https://www.digital.gov.au)

### Observed outcome

The "Explore" link in the main navigation triggers a sub-navigation list (set up manually using - as there is no current ability to add sub menu items to the main navigation menu) 

Within this list there are nested lists, representing the site structure. The pages that sit under the "Digital Experience Policy" page have been visually styled to indicate that they are child pages (level 3); however, this relationship is not marked up (and appears to be no ability to mark up) in the code or within the editorial interface. 

![Picture39](https://github.com/user-attachments/assets/4cd89f51-6058-42f7-965d-64e83821aa42)
 
### Code used
```
<ul class="ct-menu ct-menu__sub-menu ct-menu--level-1">
<li class="ct-menu__item ct-menu__item--level-1 ct-menu__item--has-children ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy" title="Policy">
Policy
</a>
<div class="ct-menu__sub-menu__wrapper ct-menu__sub-menu__wrapper--level-2">
<ul class="ct-menu ct-menu__sub-menu ct-menu--level-2">
<li class="ct-menu__item ct-menu__item--level-2 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy/ai" title="Artificial intelligence">
Artificial intelligence
</a>
</li>
<li class="ct-menu__item ct-menu__item--level-2 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy/digital-experience" title="Digital Experience Policy">
Digital Experience Policy
</a>
</li>
<li class="ct-menu__item ct-menu__item--level-2 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy-standards/standards/digital-service-standard" title="- Digital Service Standard">
- Digital Service Standard
</a>
</li>
<li class="ct-menu__item ct-menu__item--level-2 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy/digital-experience/digital-access-standard" title="- Digital Access Standard">
- Digital Access Standard
</a>
</li>
… 
</ul>
</div>
</li>
… 
</ul>
```
### Why this matters

Screen reader users may not realise that the list represents the level 3 navigation. They may not have access to the information conveyed visually.

### Expected outcome

Ability to be able to mark up the list so it is nested within the list for the level 2 navigation links. 

Preferably, use the aria-hidden attribute to hide the dash that precedes the level 3 links so that they are not announced by screen readers. 

### Code example

```
<div class="ct-menu__sub-menu__wrapper ct-menu__sub-menu__wrapper--level-2">
<ul class="ct-menu ct-menu__sub-menu ct-menu--level-2">
<li class="ct-menu__item ct-menu__item--level-2 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy/ai" title="Artificial intelligence">
Artificial intelligence
</a>
</li>
<li class="ct-menu__item ct-menu__item--level-2 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy/digital-experience" title="Digital Experience Policy">
Digital Experience Policy
</a>
</li>
<ul class="level-3">
<li class="ct-menu__item ct-menu__item--level-3 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy-standards/standards/digital-service-standard" title="- Digital Service Standard">
<span aria-hidden="true">
-
</span>
Digital Service Standard 
</a>
</li>
<li class="ct-menu__item ct-menu__item--level-2 ">
<a class="ct-link ct-theme-light ct-menu__item__link" href="/policy/digital-experience/digital-access-standard" title="- Digital Access Standard">
<span aria-hidden="true">
-
</span>
Digital Access Standard 
</a>
</li>
…
</ul>
… 
</ul>
</div>
```

**Comment by fionamorrison23** [comment]
@CJE001  Closing this issue as this is not a CivicTheme core issue. This is specifically related to the manual work done in this site.
